topology/probes/netlink.go

```diff
@@ -437,6 +460,35 @@ func (u *NetNsNetLinkProbe) addLinkToTopology(link netlink.Link) {
 	u.handleIntfIsVeth(intf, link)
 }
 
+func (u *NetNsNetLinkProbe) getRoutingTable(link netlink.Link) *RoutingTable {
+	routeList, err := u.handle.RouteList(link, netlink.FAMILY_ALL)
+	if err == nil && len(routeList) > 0 {
+		var routeTable = RoutingTable{Id: routeList[0].Table, Src: routeList[0].Src}
+		var routes []Route
+		for _, route := range routeList {
+			var r = Route{}
+			if route.Dst != nil {
+				r.Prefix = (*route.Dst).IP
```

Comment:
We need to have the mask for the prefix, route.Dst.Mask ?

Reply:
done